group_vars/all.yml

```diff
@@ -21,6 +21,10 @@ go:
   path: "{{ tmp_dir }}/go"
   module111: "on"
 
+# zfs
+zfs_image_size: 50G # Update according to server
```

Comment:
Maybe we should use an external variable here and default it to 50, just like snapshot url

Reply:
Ya mentioned it in the readme

Reply:
Pass it as extra variable